spec/array_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "./lib/array.rb"
+
+describe Array do
+	it "add all the values in array [1, 2, 3, 4]" do
+		expect([1, 2, 3, 4].rekt { |sum, number| sum + number }).to eq(10)
+	end
+	it "multiply all the values in  array [1, 2, 3, 4]" do
+    expect([1, 2, 3, 4].rekt { |sum, number| sum * number }).to eq(24)
+  end
```

Comment:
`end` at 9, 2 is not aligned with `it "multiply all the values in  array [1, 2, 3, 4]" do` at 7, 1